src/m_misc.c

```diff
@@ -2749,7 +2749,7 @@ boolean M_ParseOption(const char *p, boolean wad)
   //jff 3/3/98 skip lines not starting with an alphanum
   // killough 10/98: move to be made part of main test, add comment-handling
 
-  if (sscanf(p, "%79s %99[^\n]", name, strparm) != 2 || !isalnum(*name) ||
+  if (sscanf(p, "%79s %255[^\n]", name, strparm) != 2 || !isalnum(*name) ||
```

Comment:
@fabiangreffrath I'm tired of this code, also `256` is not enough for a path in modern OSes. What if we rewrite the config parser using `u_scanner` (not in this PR)?

Reply:
Sure, the config code is archaic. Remember when it used ints to store char pointers? 

I agree we should use the opportunity after the next point release (due this week) to clean up the config system and menu a bit.